src/components/VaccineProgressBar/VaccineProgressBar.style.tsx

```diff
@@ -3,6 +3,8 @@ import { COLOR_MAP } from 'common/colors';
 import { materialSMBreakpoint } from 'assets/theme/sizes';
 
 export const ProgressBarContainer = styled.div`
+  /* Adding display:flex makes sure the ParentSize container is tight to its contents */
+  display: flex;
```

Comment:
i think this also needs justify-content:center now. the alignment is off on mobile -

![Screen Shot 2021-06-23 at 11 32 03 AM](https://user-images.githubusercontent.com/44076375/123126066-1402db80-d417-11eb-9445-e6d2cc76c274.png)